### PR TITLE
A bunch of donut 3 improvements

### PR DESCRIPTION
**Pull request by Camryn-Buttes**
[MAPPING][QOL]
## About the PR 

Makes some needed additions to donut 3

- Adds an oxygen canister to the armory
- Adds a box of tracking implants to security
- Adds an artistic toolbox to the chaplain's office (TO BE CLEAR: this is the toolbox full of paint that spawns in the chapel on manta, oshan, destiny and I think a couple other maps. This is NOT the memetic murder toolbox)
- Adds two biosuit lockers to the pathology lab lobby
- Adds a box of pens to artlab
- Adds a costume eyepatch to the costume wardrobe in the chapel
- Adds a welding supply locker to the research podbay
- Adds a GTM to the maintenance hall south of engineering
- Adds an enucleation spoon to the medical booth
- Adds office supply wall cabinets to the following locations: the nerd dungeon, the public market stall, the bridge, and the qm's office (the little carpeted room they have)
- Adds a lighter and propuff pack to the detective's office
- Adds a portable camera monitor to the detective's office
- Adds a secure briefcase to the detective's office
- Adds a forensics cart to the detective's office, with the following items spawning inside: a box of body bags, two evidence boxes, a luminol spray bottle, an audio recorder, two tapes for the audio recorder, and a hand labeler 

## Why's this needed? 

donut 3 improvements

## Changelog

```changelog
(u)Camryn Buttes
(+)Made a bunch of minor improvements and additions to Donut 3.
```

### CODE REVIEW

File: maps/oshan.dmm [hunk 1]
```diff
@@ -31724,7 +31724,7 @@
 /obj/item/storage/box/crayon{
 	pixel_x = -6
 	},
-/obj/item/postit_stack,
+/obj/item/item_box/postittit,
```
Comment (TTerc) [inline maps/oshan.dmm:31727]:
```suggestion
/obj/item/item_box/postit,
```

File: maps/oshan_xmas.dmm [hunk 2]
```diff
@@ -33496,7 +33496,7 @@
 /obj/item/storage/box/crayon{
 	pixel_x = -6
 	},
-/obj/item/postit_stack,
+/obj/item/item_box/postittit,
```
Comment (TTerc) [inline maps/oshan_xmas.dmm:33499]:
```suggestion
/obj/item/item_box/postit,
```

File: maps/pamgoc.dmm [hunk 3]
```diff
@@ -62862,7 +62862,7 @@
 	pixel_x = -7;
 	pixel_y = 5
 	},
-/obj/item/postit_stack{
+/obj/item/item_box/postittit{
```
Comment (TTerc) [inline maps/pamgoc.dmm:62865]:
```suggestion
/obj/item/item_box/postit{
```

File: maps/pamgoc.dmm [hunk 4]
```diff
@@ -63287,7 +63287,7 @@
 	name = "autoname - SS13";
 	tag = ""
 	},
-/obj/item/postit_stack{
+/obj/item/item_box/postittit{
```
Comment (TTerc) [inline maps/pamgoc.dmm:63290]:
```suggestion
/obj/item/item_box/postit{
```

File: maps/pamgoc.dmm [hunk 5]
```diff
@@ -64980,7 +64980,7 @@
 	pixel_x = 4;
 	pixel_y = 7
 	},
-/obj/item/postit_stack{
+/obj/item/item_box/postittit{
```
Comment (TTerc) [inline maps/pamgoc.dmm:64983]:
```suggestion
/obj/item/item_box/postit{
```

**Review by TTerc — CHANGES_REQUESTED** [review]

**Comment by Camryn-Buttes** [comment]
Okay, so I'm trying to merge the suggestions, but it's just throwing an error at me and saying the diff was recently updated and I need to refresh and try again

**Comment by geneva-breaker** [comment]
why is this not two PRs

**Comment by Camryn-Buttes** [comment]
> why is this not two PRs

Because the old sticky notes kept spawning boxes of new sticky notes on the same tile as the office supply wall cabinets that I added. It was really ugly, so I had to address the issue in SOME way. 

I figured I probably shouldn't half-ass it, so I went ahead and replaced *every* instance of the old sticky notes, except for their definition.

**Comment by geneva-breaker** [comment]
> I figured I probably shouldn't half-ass it, so I went ahead and replaced _every_ instance of the old sticky notes, except for their definition.

right, but why is this in the same PR as your assorted donut fixes. 
this puts me in an awkward position of wanting to merge the stickynote fix, but being unable to do so because I'm not trusting "some other miscellaneous improvements" without review. 

And i'm not reviewing Donut3 because it's not a map i'm intimate with.

**Comment by Camryn-Buttes** [comment]
> > I figured I probably shouldn't half-ass it, so I went ahead and replaced _every_ instance of the old sticky notes, except for their definition.
> 
> right, but why is this in the same PR as your assorted donut fixes.
> this puts me in an awkward position of wanting to merge the stickynote fix, but being unable to do so because I'm not trusting "some other miscellaneous improvements" without review.
> 
> And i'm not reviewing Donut3 because it's not a map i'm intimate with.

I can provide an exhaustive list of changes later today after I manually push the fix for those postit typos.

**Comment by Camryn-Buttes** [comment]
Actually wait, I could just split it into two

**Comment by Camryn-Buttes** [comment]
There we go, reverted the postit changes, will be making a separate PR for them

Also, will write up an exhaustive list of changes in a bit.

**Comment by geneva-breaker** [comment]
cool and good thank you

**Comment by Camryn-Buttes** [comment]
> 
> 
> cool and good thank you

https://github.com/goonstation/goonstation/pull/4115

there we go

**Comment by Camryn-Buttes** [comment]
and there's the exhaustive list of changes

**Comment by Camryn-Buttes** [comment]
Hrrrgggg fukken merge conflicts

I'll fix em today

**Review by ZeWaka — APPROVED** [review]

**Review by flrsh — APPROVED** [review]

**Review by flrsh — APPROVED** [review]

**Pull request merged**